### PR TITLE
Handle remainder in distribute fee 

### PR DESCRIPTION
**Pull request by Anurupa03**

### CODE REVIEW

File: core-contracts/FeeDistribution/src/main/java/finance/omm/score/fee/distribution/AbstractFeeDistribution.java [hunk 1]
```diff
@@ -51,6 +53,9 @@ protected void distributeFee(BigInteger amount) {
                 accumulatedFee.set(receiver, feeAccumulatedAfterClaim.add(amountToDistribute));
             }
         }
+        if (remaining.compareTo(BigInteger.ZERO) > 0){
```
Comment (goldworm-icon) [inline core-contracts/FeeDistribution/src/main/java/finance/omm/score/fee/distribution/AbstractFeeDistribution.java:56]: IMO, BigInteger.signum() looks better than this.
Is there any reason why BigInteger.compareTo() is used?

```suggestion
        if (remaining.signum() > 0){
```

Comment (Anurupa03) [reply core-contracts/FeeDistribution/src/main/java/finance/omm/score/fee/distribution/AbstractFeeDistribution.java:56]: We always use BigInteger.compareTo() in our code for consistency. While we could use signum(), we stick with compareTo() to keep things uniform. 
But as curiosity is there are any discernible differences in terms of performance or step cost between the two methods.

**Review by Suyog007 — APPROVED** [review]
Looks Good.

File: core-contracts/FeeDistribution/src/main/java/finance/omm/score/fee/distribution/AbstractFeeDistribution.java [hunk 2]
```diff
@@ -52,6 +53,9 @@ protected void distributeFee(BigInteger amount) {
                 accumulatedFee.set(receiver, feeAccumulatedAfterClaim.add(amountToDistribute));
             }
         }
+        if (remaining.signum() > 0){
+            call(Contracts.sICX, "transfer", daoFundAddr, remaining);
```
Comment (goldworm-icon) [inline core-contracts/FeeDistribution/src/main/java/finance/omm/score/fee/distribution/AbstractFeeDistribution.java:57]: Is this line ok?
In line 48, "amountToDistribute" is added to collectedFee, but the remaining is not.

**Pull request merged**